clang/test/CIR/CodeGen/basic.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o - | FileCheck %s
+
+int f1();
```

Comment:
Silly question: What is the purpose here for the forward declaration of f1?

Reply:
Probably no purpose. It was in the incubator test that I'm starting to bring over here. I had taken it out, but I had some problems during development so I put it back in to see if that helped. The problem turned out to be related to handling of C versus C++ code, and I forgot to remove it after I straightened that out.